TeamCode/src/main/java/org/firstinspires/ftc/teamcode/utils/units/Vec2.kt

```diff
@@ -14,12 +14,13 @@ import kotlin.math.sqrt
  * @see Angle
  * @see Color
  */
-data class Vec2(var x: Double, var y: Double) {
+data class Vec2(@JvmField var x: Double, @JvmField var y: Double) {
     companion object {
         val ZERO = Vec2(0.0, 0.0)
     }
 
     constructor(rrVec: Vector2d) : this(rrVec.x, rrVec.y);
+    constructor(x: Double) : this(x, x)
 
     fun length() = x * x + y * y
```

Comment:
@tikhonsmovzh **[Критично]** исправить определение длины вектора
```suggestion
    fun length() = sqrt(x * x + y * y)
```